core-plugins/docs/OracleExport-action.md

```diff
@@ -0,0 +1,94 @@
+# Oracle Export Action
+
+
+Description
+-----------
+A Hydrator Action plugin to efficiently export data from Oracle to HDFS or local file system. The plugin uses
+Oracle's command line tools to export data. The data exported from this tool can then be used in Hydrator pipelines.
+
+
+Use Case
+--------
+A Hydrator user would like to export oracle data onto HDFS or local file system using an action plugin
+that exports Oracle data using the Oracle bulk export tool as opposed to using JDBC.
+
+
+Properties
+----------
+
+**oracleServerHostname:** Hostname of the remote Oracle Host.
+
+**oracleServerSSHPort:** Port to use to SSH to the remote Oracle Host. Defaults to 22.
+
+**oracleServerUsername:** Username to use to connect to the remote Oracle Host via SSH.
```

Comment:
This should probably be `oracleServerSSHUsername` instead, as it has nothing to do with credentials for Oracle DB, itself.